cdk.tf/vercel.json

```diff
@@ -178,6 +178,10 @@
     {
       "source": "/registry-providers",
       "destination": "https://registry.terraform.io/browse/providers"
+    },
+    {
+      "source": "/crash-reporting",
+      "destination": "https://developer.hashicorp.com/cdktf/telemetry#crash-reporting"
```

Comment:
The destination link does not exist yet

Reply:
that's fair --> maybe I'll just change it back and we can tackle it when we actually launch devdot?

Reply:
just noticed that this does not go into main 😁
So maybe let's add a draft PR for `dev-portal` -> `main` in which we can add a checklist to confirm the links work first?

Reply:
ok got it! I'll open a PR so we can test this out!